src/main/java/com/fabriccommunity/spookytime/client/render/TinyPumpkinRenderer.java

```diff
@@ -15,25 +17,25 @@
 	@Override
 	public void render(TinyPumpkinBlockEntity pumpkin, double x, double y, double z, float delta, int breakingStage) {
 		ItemRenderer renderer = MinecraftClient.getInstance().getItemRenderer();
-
+		
```

Comment:
Yeah expect eclipse to revert this kind of fix super quick

Reply:
IDEA will also remove these by default (iirc the setting is something like code style>java>keep indentation on empty lines)

Reply:
eh, whatever. this whole pr was automatically done with IDEA's Reformat Code, so it doesn't matter too much if bits and pieces get undone, it's pretty simple to reformat again. I might disable this option next time.